packages/form_bloc/lib/src/blocs/form/form_state.dart

```diff
@@ -20,17 +20,9 @@ abstract class FormBlocState<SuccessResponse, FailureResponse>
 
   bool isValid([int? step]) {
     if (step == null) {
-      if (_isValidByStep.isEmpty) {
```

Comment:
Of course the implementation of `every` is as follows, so that check is superfluous
```dart
bool every(bool test(E element)) {
    for (E element in this) {
      if (!test(element)) return false;
    }
    return true;
  }
```